Dockerfile

```diff
@@ -21,30 +21,30 @@ SHELL [ "/usr/bin/scl", "enable", "devtoolset-9"]
 
 # Install Tessaract
 RUN set -ex && \
-    $sudo yum install -y opencv opencv-devel opencv-python perl-core clang libpng-devel libtiff-devel libwebp-devel libjpeg-turbo-devel git-core libtool pkgconfig xz && \
-    wget https://github.com/DanBloomberg/leptonica/releases/download/1.75.1/leptonica-1.75.1.tar.gz && \
-    tar -xzvf leptonica-1.75.1.tar.gz && \
-    cd leptonica-1.75.1 || exit && \
-    ./configure && make && $sudo make install && \
-    cd .. && \
-    wget http://mirror.squ.edu.om/gnu/autoconf-archive/autoconf-archive-2017.09.28.tar.xz && \
-    tar -xvf autoconf-archive-2017.09.28.tar.xz && \
-    cd autoconf-archive-2017.09.28 || exit && \
-    ./configure && make && $sudo make install && \
-    $sudo cp m4/* /usr/share/aclocal && \
-    cd .. && \
-    git clone --depth 1  https://github.com/tesseract-ocr/tesseract.git tesseract-ocr && \
-    cd tesseract-ocr || exit && \
-    export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig && \
-    scl enable devtoolset-9 -- sh -c './autogen.sh && ./configure && make && make install' && \
-    cd .. && \
-    git clone https://github.com/tesseract-ocr/tessdata.git  && \
-    $sudo cp tessdata/*.traineddata /usr/local/share/tessdata && \
-    $sudo rm -rf /tesseract-ocr /tessdata /autoconf-archive-2017.09.28* /leptonica-1.75.1* && \
-    $sudo yum -y remove opencv opencv-devel opencv-python perl-core clang libpng-devel libtiff-devel libwebp-devel libjpeg-turbo-devel git-core libtool && \
-    $sudo rm -rf /var/cache/yum/* && \
-    $sudo rm -rf /tmp/* && \
-    yum clean all
+  $sudo yum install -y opencv opencv-devel opencv-python perl-core clang libpng-devel libtiff-devel libwebp-devel libjpeg-turbo-devel git-core libtool pkgconfig xz && \
```

Comment:
can this whitespace diff be undone? (for the sake of de-cluttering git blame and this PR)

Reply:
Yep, that was my autoformatter.